lib/xmlSchemaFaker.js

```diff
@@ -47,7 +47,7 @@ function convertSchemaToXML(name, schema, attribute, indentChar, indent) {
         attributes.push(`${key}="${propVal}"`);
       }
       else {
-        childNodes += propVal;
+        childNodes += propVal === undefined ? '' : propVal;
```

Comment:
@aman-v-singh We can be more thorough here by handling other unwanted types as well.

```
childNodes += _.isString(propVal) ? propVal : '';
```

Reply:
@VShingala Made the [changes](https://github.com/postmanlabs/openapi-to-postman/pull/679/commits/b55930904c861eb414fd3913a4087a4cce1924b4).